src/Microsoft.AspNetCore.Mvc.Formatters.Xml/SerializableErrorWrapper.cs

```diff
@@ -14,6 +14,9 @@ namespace Microsoft.AspNetCore.Mvc.Formatters.Xml
     [XmlRoot("Error")]
     public sealed class SerializableErrorWrapper : IXmlSerializable, IUnwrappable
     {
+        // Silly spelling of empty1111 to avoid collisions with other ModelState entries.
+        private static readonly string EmptyKey = "\x00C9m\x00FEt\x00DD1111";
```

Comment:
If we're actually going to be serializing this down to XML it might be valuable for the empty key to have a reference to MVC so it's known where the value is coming from.

Reply:
The serialization isn't inspected in most cases -- just deserialized. But, I'll try `"MVÇ-Émþtÿ"` out. That has the advantage of also not being a valid C# identifier (due to the `'-'`).

Reply:
Wouldn't a user-client possibly see this string? Should this be something more meaningful?

Reply:
What's "more meaningful" to you? My aim here is to make the element name mostly-readable but obviously distinct from other elements in the XML.